### PR TITLE
Prefetch, lower prefetch request priority

### PR DESCRIPTION
**Pull request by emcelyea-physna**

### CODE REVIEW

File: packages/next/src/shared/lib/router/router.ts [hunk 1]
```diff
@@ -520,6 +524,7 @@ function fetchNextData({
         isPrefetch && hasMiddleware ? { 'x-middleware-prefetch': '1' } : {}
       ),
       method: params?.method ?? 'GET',
+      priority
```
Comment (cvbuelow) [inline packages/next/src/shared/lib/router/router.ts]: i would just set it here
```suggestion
      priority: isPrefetch ? 'low' : 'auto'
```

File: packages/next/src/shared/lib/router/router.ts [hunk 2]
```diff
@@ -465,6 +465,8 @@ function fetchRetry(
     headers: Object.assign({}, options.headers, {
       'x-nextjs-data': '1',
     }),
+    // @ts-expect-error, this is newer fetch argument
```
Comment (cvbuelow) [inline packages/next/src/shared/lib/router/router.ts:468]: shouldn't need this if you update `options: Pick<RequestInit, 'method' | 'headers' | 'priority'>` above

Comment (emcelyea-physna) [reply packages/next/src/shared/lib/router/router.ts:468]: I tried that, it gets mad about `RequestInit` not having that property 😓

Comment (cvbuelow) [reply packages/next/src/shared/lib/router/router.ts:468]: huh strange, works fine for me locally

**Review by cvbuelow — APPROVED** [review]

**Pull request merged**